extension/data.ts

```diff
@@ -63,40 +105,49 @@ class RcxDict {
     return RcxDict.instance;
   }
 
-  init(loadNames) {
-    const started = +new Date();
+  static createDefaultDictEntry(): DictEntryData {
```

Comment:
Try to remove this by making one at module scope and deriving type from it.

Reply:
Some values were optionally null but I just changed their defaults to falsy string/number which should work.